src/main/java/org/hisp/dhis/tasks/GetHeavyAnalyticsTask.java

```diff
@@ -0,0 +1,87 @@
+package org.hisp.dhis.tasks;
+
+import static com.google.api.client.http.HttpStatusCodes.STATUS_CODE_OK;
+import static io.restassured.RestAssured.given;
+import static java.lang.String.format;
+import static net.andreinc.mockneat.unit.objects.From.from;
+
+import java.util.List;
+
+import org.hisp.dhis.cache.EntitiesCache;
+import org.hisp.dhis.cache.Program;
+
+import io.restassured.response.Response;
+
+/**
+ * @author Maikel Arabori <maikelarabori@gmail.com>
+ */
+public class GetHeavyAnalyticsTask
+    extends
+    DhisAbstractTask
+{
+    /**
+     * Configurable constants
+     */
+    private static final String QUERY_PARAMS = "?filter=ou:%s&dimension=dx:AUqdhY4mpvp;EY8gsfEomc0;EoYar8UxddG;GQHKiAe3DHR;GfYjGL16D0W;JIVMtpjVZqJ;Lzg9LtG1xg3;MkXHDfrFRKV;OdiHJayrsKo;Q3M7Htpzg1Y;RdkKj1rVp8R;ReUHfIn0pTQ;Rigf2d2Zbjp;Tt5TAvdfdVK;ULW9dQOiiTS;Uvn6LCg7dVU;X2XfzgH4NOR;X3taFC1HtE5;btgZ1oeF9pJ;c8fABiNpT0B;cTB6uRdmxUB;dwEq7wi6nXV;elTqMUTRgdk;fLbrQ4jEtn6;iug406zdfTE;lI5Q5vkydYu;lOiynlltFdy;lZZxDlIsvTc;n0GE1ISYrdM;nfG18MJZX5o;nkjlWUMIdHh;oTOpKabJA1v;puykO1tbcdi;qmCKRmVj4WX;ryFBhJdetdM;sB79w2hiLp8;sMTMkudvLCD;tcs5YGnjiKo;ulgL07PF8rq;vKWOc4itBo2;vihpFUg2WTy&dimension=pe:LAST_12_MONTHS&displayProperty=NAME&skipMeta=true&includeNumDen=true";
```

Comment:
Instead of this big String, can you do something like:
```
private final List<String> dimensions = Lists.newArrayList("AUqdhY4mpvp", "EY8gsfEomc0",
            "EoYar8UxddG", "GQHKiAe3DHR", ...);

// Warm-up
        given().queryParam("filter", "ou")
               .queryParam("dimension", Joiner.on(";").join(dimensions))
               .queryParam("dimension", "pe:LAST_12_MONTHS")
               .queryParam("skipMeta", "true")
                .get( query( theParams ) );
```

Reply:
Done. Cheers!